src/services/submissionService.ts

```diff
@@ -0,0 +1,62 @@
+import gql from 'graphql-tag'
+import { Submission } from '../typeDefs/dbSchema'
+import { DocumentData, WhereFilterOp, getFirestore } from 'firebase-admin/firestore'
+
+export const getSubmissionById = async (id: string) : Promise<Submission | null> => {
+    const db = getFirestore()
+    const submissionRef = db.collection('submissions')
+    const whereCondition = '=' as WhereFilterOp
+    const snapshot = await submissionRef.where('id', whereCondition, id).get()
+
+    if (snapshot.docs.length < 1) {
+        return null
+    }
+
+    const convertedEntity = mapDbEntityTogqlEntity(snapshot.docs[0].data())
+
+    return convertedEntity
+}
+
+export const getSubmissions = async (userSearchQuery: string[]) : Promise<Submission[]> => {
+    const db = getFirestore()
+    const submissionRef = db.collection('submissions')
+    const snapshot = await submissionRef.where('id', 'in', userSearchQuery).get()
+
+    const submissions = [] as Submission[]
+
+    snapshot.forEach(doc => {
+        const convertedEntity = mapDbEntityTogqlEntity(doc.data())
+
+        submissions.push(convertedEntity)
+    })
+
+    return submissions
+}
+
+export const addSubmission = async (submission: Submission) : Promise<void> => {
+    const db = getFirestore()
+    const submissionRef = db.collection('submissions')
+
+    await submissionRef.add(submission)
```

Comment:
we should actually send back a response for the client to know if it was successful. 
I can do this later 👍

Reply:
Thanks for the feedback, Philip! I can add this tomorrow. I'm thinking a try/catch block would be good as it would include error handling.